chris_backend/pipelines/tests/test_views.py

```diff
@@ -130,7 +130,6 @@ def test_pipeline_list_success(self):
 
     def test_pipeline_list_failure_unauthenticated(self):
         response = self.client.get(self.create_read_url)
-        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
instead of deleting this line, maybe we can instead assert that the status is 200 OK?